src/Shared/Microsoft.AspNetCore.Razor.Serialization.Json/ObjectWriters.cs

```diff
@@ -35,6 +35,8 @@ public static void WriteProperties(JsonDataWriter writer, RazorConfiguration val
 
         writer.Write(nameof(value.LanguageVersion), languageVersionText);
 
+        writer.WriteIfNotFalse(nameof(value.SuppressAddComponentParameter), value.SuppressAddComponentParameter);
```

Comment:
Should `LanguageServerFlags` and `UseConsolidatedViews` also be serialized?

Reply:
LanguageServer flags no, because the roslyn side will never know a correct value for that. UseConsolidatedViews I'm unclear on. I would guess yes? But I need to dig into it more.